db/migrate/20210407185420_create_push_notification_targets.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePushNotificationTargets < ActiveRecord::Migration[6.1]
+  def change
+    create_table :push_notification_targets do |t|
```

Comment:
shouldn't this have a FK to `devices` ?

Reply:
I refactored this to make sure we have this FK to avoid any duplication and hoping to keep referential integrity. This means the `app_bundle` is now part of what's now `AppIntegration` and devices are now referencing them. 

`app_bundle` was part of of a composite unique index in `Devices` so it became a bit harder to refactor with the migrations but I think it came out okay, let me know if this is what you had in mind or if it works alright 🙂